pom.xml

```diff
@@ -14,7 +14,7 @@
         <java.version>1.8</java.version>
         <scala.version>2.11.8</scala.version>
         <scala.binary.version>2.11</scala.binary.version>
-        <rug.version>(0.25.0,1.0.0-m.5]</rug.version>
+        <rug.version>(0.25.0,1.0.0-m.6]</rug.version>
```

Comment:
You're probably right. This range scheme came before I looked at this lib and I'm not sure why it is like this. I'll update it